lib/pure/algorithm.nim

```diff
@@ -46,6 +46,8 @@
 ## * `sequtils module<sequtils.html>`_ for working with the built-in seq type
 ## * `tables module<tables.html>`_ for sorting tables
 
+import std/private/since
```

Comment:
followup on https://github.com/nim-lang/Nim/pull/16182#issuecomment-735619108 can you rename the old private algorithm.merge to algorithm.mergeAlt ?

Reply:
I could make a follow up PR to rename it. Then we could talk about the new name for it.

Reply:
ok; let's keep this comment open then